tests/default_vars.sh

```diff
@@ -1453,3 +1454,131 @@ export CPLMODE=hafs
 export RUNTYPE=startup
 export USE_COLDSTART=false
 }
+
+export_hafs ()
+{
+export FV3=true
+export S2S=false
+export HAFS=true
+export DATM_CDEPS=false
+export DOCN_CDEPS=false
+export THRD=1
+export INPES=$INPES_dflt
+export JNPES=$JNPES_dflt
+export TASKS=$TASKS_dflt
+export TPN=$TPN_dflt
+
+export OUTPUT_GRID='regional_latlon'
+export IMO=-999999
+export JMO=-999999
+export CEN_LON=-999.99
+export CEN_LAT=-999.99
+export LON1=-999.99
+export LAT1=-999.99
+export LON2=-999.99
+export LAT2=-999.99
+export DLON=-999.99
+export DLAT=-999.99
+export STDLAT1=-999.99
+export STDLAT2=-999.99
+export NX_LC=-999999
+export NY_LC=-999999
+export DX_LC=-999.99
+export DY_LC=-999.99
+
+export OUTPUT_GRID_2='regional_latlon'
+export IMO_2=-999999
+export JMO_2=-999999
+export CEN_LON_2=-999.99
+export CEN_LAT_2=-999.99
+export LON1_2=-999.99
+export LAT1_2=-999.99
+export LON2_2=-999.99
+export LAT2_2=-999.99
+export DLON_2=-999.99
+export DLAT_2=-999.99
+export STDLAT1_2=-999.99
+export STDLAT2_2=-999.99
+export NX_LC_2=-999999
+export NY_LC_2=-999999
+export DX_LC_2=-999.99
+export DY_LC_2=-999.99
```

Comment:
These are not good default values. In fact there should be no default value at all, they should be left undefined, since each test's configure file already exports all these variables explicitly.

Reply:
@DusanJovic-NOAA The reason I defined these for HAFS RTs is because I want to use one model_configure template, e.g., [tests/parm/model_configure_hafs_shared.IN](https://github.com/DusanJovic-NOAA/ufs-weather-model/pull/19/files#diff-bade0368dd490faecfbb0364e9ce59ee3b1261e2135d9b1e45f5a6a8678e8ac6) to support all HAFS RTs (single or multiple output grids). Of course one can use separate model_configure templates for different RTs, but for HAFS RTs, we would like avoid such situation if possible. And the "-999999" or "-999.99" are just chosen arbitrary, and they can just be whatever you want at this point. Also, as you mentioned in another email thread, they are not actually used/parsed if the corresponding output grid options were not chosen.
Also, appreciate it if you could suggest another solution if this approach does not sound good to you.
Thanks!

Reply:
Just define them as empty stings here.